js/templates/modals/settings/advanced.html

```diff
@@ -0,0 +1,298 @@
+          </div>
+        </div>
+      </div>
+      <div class="flexRow">
+        <div class="col3">
+          <label><%= ob.polyT('settings.advancedTab.transaction.defaultFee') %></label>
+        </div>
+        <div class="col9">
+          <div class="btnRadio clrBr">
+            <input type="radio"
+                   persist="local"
+                   name="defaultTransactionFee"
+                   value="low"
+                   id="defaultTransactionFeeLow"
+                   data-var-type="string"
+            <% if(ob.defaultTransactionFee == "low") { %>checked<% } %>>
+            <label for="defaultTransactionFeeLow"><%= ob.polyT('settings.advancedTab.transaction.low') %></label>
+          </div>
+          <div class="btnRadio clrBr">
+            <input type="radio"
+                   persist="local"
+                   name="defaultTransactionFee"
+                   value="medium"
+                   id="defaultTransactionFeeMedium"
+                   data-var-type="string"
+            <% if(ob.defaultTransactionFee == "medium") { %>checked<% } %>>
+            <label for="defaultTransactionFeeMedium"><%= ob.polyT('settings.advancedTab.transaction.medium') %></label>
+          </div>
+          <div class="btnRadio clrBr">
+            <input type="radio"
+                   persist="local"
+                   name="defaultTransactionFee"
+                   value="high"
+                   id="defaultTransactionFeeHigh"
+                   data-var-type="string"
+            <% if(ob.defaultTransactionFee == "high") { %>checked<% } %>>
+            <label for="defaultTransactionFeeHigh"><%= ob.polyT('settings.advancedTab.transaction.high') %></label>
+          </div>
+        </div>
+      </div>
+    </form>
+  </div>
+  <div class="rowMd border clrBr js-serverContainer">
+    <form class="box padKids padStack clrP clrBr js-server">
+      <div class="flexRow">
+        <h3><%= ob.polyT('settings.advancedTab.server._sectionName') %></h3>
+      </div>
+      <div class="flexRow">
+        <div class="col3">
+          <label>
+            <%= ob.polyT('settings.advancedTab.server.management') %>
+          </label>
+        </div>
+        <div class="col9">
+          <a class="btn clrP clrBr js-showConnectionManagement flexNoShrink"><%= ob.polyT('settings.advancedTab.server.btnShowManagementAction') %></a>
+        </div>
+      </div>
+      <div class="flexRow">
+        <div class="col3">
+          <label>
+            <%= ob.polyT('settings.advancedTab.server.peers') %>
+          </label>
+        </div>
+        <div class="col9">
+          <a class="btn clrP clrBr js-showConnectedPeers flexNoShrink"><%= ob.polyT('settings.advancedTab.server.btnShowPeersAction') %></a>
+        </div>
+      </div>
+      <div class="flexRow">
+        <div class="col3">
+          <label>
+            <%= ob.polyT('settings.onOff') %>
+          </label>
+        </div>
+        <div class="col9">
```

Comment:
We've decided to remove this functionality, since the server is automatically turned off / on when you connect to the default connection (coming soon).